### PR TITLE
Enable legacy download counts.

### PR DESCRIPTION
**Pull request by frankban**
Those values are stored in the extra-info for the entity.
This is basically a hack, and will be removed soon.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/627/

**Comment by mhilton** [comment]
:+1: LGTM (No QA)

File: internal/charmstore/search_test.go [hunk 1]
```diff
@@ -27,6 +27,17 @@ var _ = gc.Suite(&StoreSearchSuite{})
 
 func (s *StoreSearchSuite) SetUpTest(c *gc.C) {
 	s.IsolatedMgoESSuite.SetUpTest(c)
+
+	// Temporarily set LegacyDownloadCountsEnabled to false, so that the real
+	// code path can be reached by tests in this suite.
+	// TODO (frankban): remove this block when removing the legacy counts
```
Comment (urosj) [inline internal/charmstore/search_test.go:33]: Made a card for this. Thanks.

**Comment by urosj** [comment]
:+1: Thank you for all the TODO notices.

PS: QA will be done when all pieces are in place.

**Comment by frankban** [comment]
Thanks for the reviews!
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**